sncosmo/spectral.py

```diff
@@ -468,16 +500,89 @@ def zpbandflux(self, band):
 
         return bandflux
 
-    def band_flux_to_mag(self, flux, band):
-        """Convert flux (photons / s / cm^2) to magnitude."""
-        return -2.5 * math.log10(flux / self.zpbandflux(band))
+class NaturalMagSystem(MagSystem):
 
-    def band_mag_to_flux(self, mag, band):
-        """Convert magnitude to flux in photons / s / cm^2"""
-        return self.zpbandflux(band) * 10.**(-0.4 * mag)
+    """A magnitude system in which the magnitudes / fluxes are
+    calibrated such that they represent what is measured through the
+    filters attached to a specific telescope (not standard filters). 
+
+    Parameters
+    ----------
+    
+    bands: iterable of `~sncosmo.Bandpass` or str
+        The filters in the magnitude system.
+    zeropoints: `~numpy.ndarray`
+        The zeropoints of the filters. 
+    standards: iterable of `~sncosmo.Spectrum`,
+        The spectrophotmetric flux standards for each band.
+    """
+
+    def __init__(self, bands, zeropoints, standards, name=None):
+        super(NaturalMagSystem, self).__init__(name)
+        if len(bands) != len(zeropoints):
+            raise ValueError, 'Number of passed bands (%d) does not equal '  \
+                'the number of passed zeropoints (%d).'  % (len(bands),      
+                                                            len(zeropoints))
```

Comment:
Minor style notes for consistency with rest of code base: (1) Instantiate exceptions with parentheses, (2) don't use `\` for line continuation, (3) use `.format` over `%`. E.g.,
```python
raise ValueError('Number of passed bands ({}) does not equal '
                 'the number of passed zeropoints({}).'
                 .format(len(bands), len(zeropoints)))